packages/kmod-5.15-nvidia/kmod-5.15-nvidia.spec

```diff
@@ -29,6 +29,7 @@ URL: http://www.nvidia.com/
 Source0: https://us.download.nvidia.com/tesla/%{tesla_ver}/NVIDIA-Linux-x86_64-%{tesla_ver}.run
 Source1: https://us.download.nvidia.com/tesla/%{tesla_ver}/NVIDIA-Linux-aarch64-%{tesla_ver}.run
 Source2: NVidiaEULAforAWS.pdf
+Source3: https://github.com/NVIDIA/open-gpu-kernel-modules/blob/535/COPYING
```

Comment:
You may have wanted to use the url https://raw.githubusercontent.com/NVIDIA/open-gpu-kernel-modules/535/COPYING

This url is full of GitHub UX scripting and html.

Reply:
I'll update to just the filename since that is consistent with the rest. The Cargo.toml `external-files` URL *is* the right path which is the important one.